WebKitBrowser/WebKitBrowser.config

```diff
@@ -41,7 +41,15 @@ map()
     kv(msebuffers ${PLUGIN_WEBKITBROWSER_MSEBUFFERS})
     kv(thunderdecryptorpreference ${PLUGIN_WEBKITBROWSER_THUNDER_DECRYPTOR_PREFERENCE})
     kv(memoryprofile ${PLUGIN_WEBKITBROWSER_MEMORYPROFILE})
-    kv(memorypressure ${PLUGIN_WEBKITBROWSER_MEMORYPRESSURE})
+    if(PLUGIN_WEBKITBROWSER_MEMORYPRESSURE)
```

Comment:
Like in the WebKitBrowser.conf.in let's also use  "if(WEBKIT_MEMORY_PRESSURE_API)" here to make more clear that only either the old or new can be used but not both

Reply:
done.